packages/normalizr/src/WeakListMap.ts

```diff
@@ -0,0 +1,65 @@
+/** Link in a chain */
+class Link<K extends object, V> {
+  children = new WeakMap<K, Link<K, V>>();
```

Comment:
What are normalizr's supported engine targets? WeakMap is not polyfillable, so you'd need to fall back to `Map` if you wanted to support engines that lacked it.

Reply:
Looks like WeakMap starts in v6, which is well beyond what most people support now. I'll add an engine entry to ensure this.

Reply:
Note that adding an engines entry is semver-major.